### PR TITLE
Reload messages on /gpreload

### PR DESCRIPTION
**Pull request by budi1200**

### CODE REVIEW

File: src/main/java/me/ryanhamshire/GriefPrevention/DataStore.java
Comment (Jikoo) [inline src/main/java/me/ryanhamshire/GriefPrevention/DataStore.java]: Should probably be protected, not public.

File: src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java
Comment (Jikoo) [inline src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java]: This line is indented with tabs, should be spaces.

File: src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java
Comment (Jikoo) [inline src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java]: Opening brace should be on the next line.

Comment (Jikoo) [inline src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java]: Should consider protected here as well.

**Review by Jikoo — CHANGES_REQUESTED** [review]
Overall looks good, definitely a useful change. Couple of style issues and visibility choices I'd argue against, but other than that 👍

**Comment by RoboMWM** [comment]
hmm, guess not all IDEs respect .editorconfig

**Comment by Jikoo** [comment]
> hmm, guess not all IDEs respect .editorconfig

Yeah, it's an IJ thing, but it's moderately portable.

**Review by Jikoo — APPROVED** [review]
Looks good! Reviewing again to dismiss my old review, you'll need Robo's approval for pulling though.

**Comment by KayaEdwin** [comment]
Could this be merged?

**Review by RoboMWM — APPROVED** [review]

**Pull request merged**
